### ISSUE TITLE
enterpriseId does not support sub-identifiers

### ISSUE THREAD

**Issue by timclassic** [jobec/rfc5424-logging-handler#34]
The `enterprise_id` parameter is currently an integer, which prevents the use of sub-identifiers.  RFC 5424 says:

```
                                    ... An enterprise might decide to
   use sub-identifiers below its private enterprise number.  If sub-
   identifiers are used, they MUST be separated by periods and be
   represented as decimal numbers.  An example for that would be
   "32473.1.2".  Please note that the ID "32473.1.2" is just an example
   and MUST NOT be used.  The complete up-to-date list of Private
   Enterprise Numbers (PEN) is maintained by IANA.
```
(link: https://tools.ietf.org/html/rfc5424#section-7.2.2)

Perhaps `enterprise_id` should instead be a string?

**Comment by timclassic** [comment]
Or perhaps it should be a list of integers?

**Comment by jobec** [comment]
Fixed in b002b5d57ddc9a6c055f003d85b195e17f51ad48
